src/plugins/dashboard/public/application/actions/library_notification_action.tsx

```diff
@@ -79,6 +79,8 @@ export class LibraryNotificationAction implements ActionByType<typeof ACTION_LIB
 
   public isCompatible = async ({ embeddable }: LibraryNotificationActionContext) => {
     return (
+      embeddable.getRoot() &&
+      embeddable.getRoot().isContainer &&
```

Comment:
Actually here, it could just be `embeddable.getRoot().isContainer` since getRoot returns the embeddable itself if there is no parent. This is a lot cleaner though!